### ISSUE TITLE
Lab Report 2 Feedback

### ISSUE THREAD

**Issue by kheravi** [Cthloveross/cse15l-lab-reports2#1]
Here is your feedback for Lab Report 2:

- [x] Correct screenshot of code diff page
- [x] Correct failure inducing test file - 1
- [x] Correct failure inducing test file - 2
- [x] Correct failure inducing test file - 3
- [ ] Screenshot of symptom output that caused the failure for test file - 1
- [ ] Screenshot of symptom output that caused the failure for test file - 2
- [ ] Screenshot of symptom output that caused the failure for test file - 3
Put screenshots of your terminal failing for each of the 3 tests as your symptom output!
- [x] Correct description for the relationship between the bug, the symptom, and the failure-inducing input for file - 1
- [x] Correct description for the relationship between the bug, the symptom, and the failure-inducing input for file - 2
- [x] Correct description for the relationship between the bug, the symptom, and the failure-inducing input for file - 3
- [x] Correct markdown formatting
- [x] Lab report is on its own page, not the index page
- [x] Text is formatted in a way that was easy for grader to read

Additional comments from your grader:
Put screenshots of your terminal failing for each of the 3 tests as your symptom and you should get a 100%!

--------

If you don't have all the checkmarks above, fix the issues described, and then submit a [regrade request on Gradescope](https://help.gradescope.com/article/8hchz9h8wh-student-regrade-request). Use this template for your regrade request:

- Link to this issue
- Link to your published lab report page

**Comment by Cthloveross** [comment]
I just changed my report. Can you help me check that? Btw, I cannot see the grade on gradescope, so I cannot submit a regrade request. But there is a late submission choice, should I submit there?

**Comment by kheravi** [comment]
Yup those all look good! Just submit a regrade request once we publish grades which should be sometime Friday!

**Comment by Cthloveross** [comment]
thank you
